test-runtime/imports/tsconfig.declarations.json

```
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "lib": [
+      "dom",
+      "es2016"
+    ],
+    "pretty": true,
+    "jsx": "react",
+    "strict": true,
+    "noEmit": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictFunctionTypes": true,
+    "strictNullChecks": true
+  },
+  "include": [
+    "../../testkit/*.d.ts"
```

```suggestion
    "../../dist/testkit/*.d.ts"
```
it should point the `dist` folder